README.md

```diff
@@ -171,11 +212,11 @@ copy&paste this to your `~/.emacs`:
 (autoload 'utop "utop" "Toplevel for OCaml" t)
 ```
 
-### Usage
+### [Usage](#usage-emacs)
```

Comment:
That's done to easily distinguish between this heading and the other "Usage" for utop itself.